tests/src/data/use-cases/db-add-guardian.spec.ts

```diff
@@ -72,19 +72,19 @@ describe('DbAddGuardian use case', () => {
       await expect(promise).rejects.toThrow()
     })
 
-    it('Should return undefined if email is already registered', async () => {
+    it('Should return null if email is already registered', async () => {
       const { sut, guardianRepositoryStub } = makeSut()
       jest.spyOn(guardianRepositoryStub, 'add').mockResolvedValue(undefined)
       const result = await sut.add(params)
-      expect(result).toBeUndefined()
+      expect(result).toBeFalsy()
     })
   })
 
   test('Should return a guardian when saving the user successfully', async () => {
     const { sut } = makeSut()
-    const result = await sut.add(params)
+    const result = await sut.add(params) as any
     expect(result).toHaveProperty('id')
-    expect(result?.id).toBeDefined()
+    expect(result.id).toBeDefined()
     expect(result).toMatchObject({
       firstName: params.firstName,
       lastName: params.lastName,
```

Comment:
Change to 
```ts
expect(result).toEqual({
	id: expect.any(String),
	firstName: params.firstName,
	lastName: params.lastName,
	email: params.email,
	phone: params.phone
})
```

Reply:
this part ( id: expect.any(String) ) replace this part ( id: expect.any(String) ) ?

Reply:
I meant to replace the `expect` in line 88 with
```ts
expect(result).toEqual({
	id: expect.any(String),
	firstName: params.firstName,
	lastName: params.lastName,
	email: params.email,
	phone: params.phone
})
```

this makes the previous `expect`, `expect(result.id).toBeDefined()` and `expect(result).toHaveProperty('id')`, redundant and can be deleted